nilmtk/elecmeter.py

```diff
@@ -421,7 +421,7 @@ def load(self, **kwargs):
             print("ElecMeter.load")
             print(self)
 
-        if 'sample_period' in kwargs:
+        if 'sample_period' in kwargs and kwargs['resample'] != False:
```

Comment:
(After writing that comment, I noticed that Travis agrees about the `KeyError` issue I mentioned!  This is from the Travis log for this pull request:

````======================================================================
ERROR: test_load (nilmtk.tests.test_metergroup.TestMeterGroup)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/nilmtk/nilmtk/nilmtk/tests/test_metergroup.py", line 141, in test_load
    df = elec.load().next()
  File "/home/travis/build/nilmtk/nilmtk/nilmtk/metergroup.py", line 685, in load
    index, columns, self.meters, kwargs)
  File "/home/travis/build/nilmtk/nilmtk/nilmtk/metergroup.py", line 1735, in combine_chunks_from_generators
    generator = meter.load(**kwargs_copy)
  File "/home/travis/build/nilmtk/nilmtk/nilmtk/elecmeter.py", line 424, in load
    if 'sample_period' in kwargs and kwargs['resample'] != False:
KeyError: 'resample'
```)